deploy/helm/README.md

```diff
@@ -0,0 +1,134 @@
+# swo-k8s-collector
+
+## Table of contents
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Limitations](#limitations)
+
+## Installation
+
+Walk through `Add Kubernetes` in [SolarWinds Observability](https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=gh-k8s-collector)
+
+
+## Configuration
+
+The [Helm chart](Chart.yaml) that you are about to deploy to your cluster has various configuration options. The full list, including the default settings, is available in [values.yaml](values.yaml).
+
+Internally, it contains [OpenTelemetry Collector configuration](https://opentelemetry.io/docs/collector/configuration/), which defines the metrics and logs to be monitored as well as their preprocessing.
+
+**WARNING: Custom modifications to OpenTelemetry Collector configurations can lead to unexpected `swo-k8s-collector` behavior, data loss, and subsequent entity ingestion failures on the Solarwinds Observability platform side.**
+
+### Mandatory configuration
+
+In order to deploy the Helm chart, you need to prepare:
+
+- A secret called `solarwinds-api-token` with API token for sending data to SolarWinds Observability
+- A Helm chart configuration:
+
+  ```yaml
+  otel:
+      endpoint: <solarwinds-observability-otel-endpoint>
+  cluster:
+      name: <cluster-display-name>
+      uid: <unique-cluster-identifier>
+  ```
+
+### Metrics
+
+By default, the `swo-k8s-collector` collects a subset of `kube-state-metrics` metrics and metrics exposed by workloads that are annotated with `prometheus.io/scrape: true`.
+
+To configure the autodiscovery, see settings in the `otel.metrics.autodiscovery.prometheusEndpoints` section of the [values.yaml](values.yaml).
+
+Once deployed to a Kubernetes cluster, the metrics collection and processing configuration is stored as a ConfigMap under the `metrics.config` key.
+
+In order to reduce the size of the collected data, the `swo-k8s-collector` collects only selected `kube-state-metrics` metrics that are key for successful entity ingestion on the SolarWinds Observability side. 
```

Comment:
Since you are already changing this paragraph, I'd probably drop it completely. While not wrong, it's a bit misleading because the autodiscovery is enabled by default.